content/300-guides/050-database/100-developing-with-prisma-migrate/168-include-unsupported-database-features.mdx

```diff
@@ -21,16 +14,23 @@ Prisma Migrate uses the Prisma schema to determine what features to create in th
 
 To add an unsupported feature to your database, you must [customize a migration](customizing-migrations) to include that feature before you apply it.
 
-<Admonition type="info">
-
 <Tip>
 
 The Prisma schema is able to represent [unsupported field types](/concepts/components/prisma-schema/features-without-psl-equivalent#unsupported-field-types) <span class="concept"></span> and [native database functions](enable-native-database-functions).
 
 </Tip>
 
+<Admonition type="info">
+
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate dev`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
+
 </Admonition>
```

Comment:
Point out that Migrate for MongoDB is db push instead of "not supported" warning.